ssntp/server.go

```diff
@@ -181,8 +181,23 @@ func handleSSNTPClient(server *Server, conn net.Conn) {
 		switch frame.Type {
 		case COMMAND:
 			if (Command)(frame.Operand) == CONFIGURE && session.destRole.IsController() {
-				/* TODO Send the CONFIGURE payload to the config package */
+				// setup new configuration in SSNTP server
 				server.configuration.setConfiguration(frame.Payload)
+
+				// save new configuration to be persistent
+				pyld, err := configuration.Payload(frame.Payload)
+				if err != nil {
+					server.log.Errorf("Unable to retrieve configuration payload: %v\n", err)
```

Comment:
If I return/break here, wouldn't the connection between ciao-scheduler and ciao-controller would be closed?
This point is reached when the ciao-controller send the `CONFIGURE` command to the scheduler, meaning that the ciao-scheduler has been initialized already and it's handling `SSNTP client` requests, that's why I didn't break/return; because at this point this configuration change is yet another SSNTP client command for the SSNTP server.